config/react-config.json

```diff
@@ -17,5 +17,14 @@
     "i18nConfigPath": "/bahmni_config/openmrs/i18n/",
     "BAHMNI_CONFIG_URL": "/bahmni_config/openmrs/apps",
     "IMPLEMENTATION_CONFIG_URL": "/implementation_config/openmrs/apps",
-    "locationTagName": "Appointment Location"
+    "locationTagName": "Appointment Location",
+    "otpServiceUrl": "/otp-service",
+    "personUrl": "/openmrs/ws/rest/v1/person/",
+    "enableAppointmentBookingSMSAlert": true,
```

Comment:
This repository is "code" repo. Hence the configurations in this file cannot be changed by users after deployment. 

We need to expose configs like `enableAppointmentBookingSMSAlert` via UI, so that during demos, Akhil/etc can just toggle this ON/OFF. Which means, it will need to be read via an Advanced Setting in OpenMRS UI.

Also, same for Clinic Name -- can't be hardcoded in "code". Needs to be an external configuration.

Reply:
All the configurable values are coming from clinic-config also. These are like default values. do we need to move all configs to read from global-properties or just the toggle variables like enableAppointmentBookingSMSAlert, enableRegistrationSMSAlert ?

Reply:
For SMS alert, we will need to read from global properties, because Akhil will want to demo them without a re-deployment. He will switch it on, demo and then switch off to save money.

For all other settings, if they can be overridden using clinic-config, then its OK i think. We don't need to do anything else. I just wanted to make sure, the values can be configured without editing any "code repo".

Reply:
Updated the toggle variables to be read from global property and left the other configurable values as clinic configuration